SimPEG/electromagnetics/frequency_domain/simulation.py

```diff
@@ -240,6 +240,84 @@ def Jtvec(self, m, v, f=None):
 
         return mkvc(Jtv)
 
+    def getJ(self, m, f=None):
+        """
+        Method to form full J given a model m
+
+        :param numpy.ndarray m: inversion model (nP,)
+        :param SimPEG.electromagnetics.frequency_domain.fields.FieldsFDEM u: fields object
+        :rtype: numpy.ndarray
+        :return: J (ndata, nP)
+        """
+
```

Comment:
```suggestion
self.model = m
```

Should set the model in case it has changed so that the sensitivity is recomputed if necessary (see #1358)